lib/pipenv.js

```diff
@@ -32,7 +32,14 @@ async function pipfileToRequirements() {
     try {
       res = await spawn(
         'pipenv',
-        ['lock', '--requirements', '--keep-outdated'],
+        ['lock', '--keep-outdated'],
```

Comment:
does it make sense to retain this command spawn if its result is anyway overridden by the new `requirements --hash` command?

Reply:
`pipenv lock` command
generates Pipfile.lock file
https://pipenv.pypa.io/en/latest/cli/#pipenv-lock

`pipenv requirements` command
requires a Pipfile.lock file to run
https://pipenv.pypa.io/en/latest/cli/#pipenv-requirements

but I guess when running this plugin the Pipfile.lock is already there on place so it should be ok to remove as you say. but I am a bit to new to python and this plugin to be 100% sure to be honest.

Reply:
i see. so perhaps it's needed if some repo lacks the lock file?

Reply:
This is a needed fix. With the deprecated --requirements param I get this error:
Error: `pipenv lock --requirements --keep-outdated` Exited with code 2

I removed this parameter on my local code and works.

Reply:
fair enough!